apps/silverback-website/docs/gatsby/packages.mdx

```diff
@@ -0,0 +1,243 @@
+pull request to add the new library to this documentation, so we can have a
+discussion about integrating it in our stack.
+
+You can use these websites to get different indicators, like popularity,
+quality, stars or download count. Is then up to you to pick the right one for
+your purpose.
+
+- [npm](https://www.npmjs.com/)
+- [Openbase](https://openbase.com/)
+
+## Fetching data
+
+### [`axios`](https://github.com/axios/axios)
+
+**Description**: Promise based HTTP client for the browser and node.js.
+
+**Features**:
+- Make XMLHttpRequests from the browser
+- Make http requests from node.js
+- Supports the Promise API
+- Intercept request and response
+- Transform request and response data
+- Cancel requests
+- Automatic transforms for JSON data
+- Client side support for protecting against XSRF
+
+### [`react-query`](https://github.com/tannerlinsley/react-query)
+
+**Description**: Hooks for fetching, caching and updating asynchronous data in
+React.
+
+**Features**:
+
+- Transport/protocol/backend agnostic data fetching (REST, GraphQL, promises,
+  whatever!)
+- Auto Caching + Refetching (stale-while-revalidate, Window Refocus,
+  Polling/Realtime)
+- Parallel + Dependent Queries
+- Mutations + Reactive Query Refetching
+- Multi-layer Cache + Automatic Garbage Collection
+- Paginated + Cursor-based Queries
+- Load-More + Infinite Scroll Queries w/ Scroll Recovery
+- Request Cancellation
+- React Suspense + Fetch-As-You-Render Query Prefetching
+
+### [`jsonapi-react`](https://github.com/aribouius/jsonapi-react)
+
+**Description**: A minimal JSON:API client and React hooks for fetching,
+updating, and caching remote data.
+
+**Features**: The JSON:API specification offers numerous benefits for writing
+and consuming REST API's, but at the expense of clients being required to manage
+complex schema serializations. This library try to mimic what `react-query`
+does, but with JSON:API endpoints.
+
+### [`isomorphic-fetch`](https://github.com/matthew-andrews/isomorphic-fetch)
+
+**Description**: Fetch for node and Browserify. Built on top of
+[GitHub's WHATWG Fetch polyfill](https://github.com/github/fetch).
+
+**Features**: The Fetch API is currently not implemented consistently across
+browsers. This module will enable you to use fetch in your Node code in a
+cross-browser compliant fashion. The Fetch API is part of the Web platform API
+defined by the standards bodies WHATWG and W3C.
+
+## Forms
+
+### [`formik`](https://github.com/formium/formik)
+
+**Description**: Build forms in React, without the tears.
+
+**Features**:
+- Form-level Validation
+- Error Messages
+- Wizards and multi-step forms
+- Field-level validation
+- Array Fields
+- API Errors
+- Custom Validation
+- Internationalization (i18n)
+- Auto-saving forms
+- Dependent Validation
+- Conditional Logic
+- Dynamic Fields
+- React Native
+- Tree shakeable
+
+## Maps
+
+### [`google-map-react`](https://github.com/google-map-react/google-map-react)
```

Comment:
This might be something that needs a little additional research. A lot of projects, especially in government, can't use Google Maps for legal reasons. I would be good to have an alternative. Currently I see two other options:

* Fully open: Leaflet + OpenStreetMap (https://react-leaflet.js.org/)
* Paid service, but not Google: MapBox + react-map-gl (https://visgl.github.io/react-map-gl/)

Largely depends on the actual requirements, so we should come up with a comparison + decision guideline.

Reply:
Yeah, I'd rather prefer to add additional libraries when needed, so we don't put too much effort into researching libraries before we actually make use of them.